### PR TITLE
Bootstrap pulp_python plugin

### PR DESCRIPTION
**Pull request by dralley**
Everything needed for getting the plugin to be installed properly by Pulp 3, and get basic CRUD functionality working.

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright © 2014 Pulp Project developers.
+Copyright © 2017 Pulp Project developers.
```
Comment (dralley) [inline COPYRIGHT]: Would it be best to leave this at 2014?  I don't think it matters one way or the other in this case

Comment (werwty) [reply COPYRIGHT]: I think this should be left as 2014: 

> The general rule is that the year to include in a copyright notice is the year of first publication of the work. [0]

[0] http://www.copyrightlaws.com/copyright-notice-year/

File: docs/Makefile [hunk 2]
```diff
@@ -1,153 +0,0 @@
-# Makefile for Sphinx documentation
```
Comment (werwty) [inline docs/Makefile]: I think eventually we're still planning on building sphinx docs. Although I am fine with this being deleted for now and bought back when we have sphinx docs to build.

File: docs/release-notes/1.0.rst [hunk 3]
```diff
@@ -1,51 +0,0 @@
-1.0 Release Notes
```
Comment (werwty) [inline docs/release-notes/1.0.rst]: Same for these. I think we will need to bring the release notes back at some point, but it's fine if they go for now.

File: pulp_python/app/models.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
+"""
```
Comment (werwty) [inline pulp_python/app/models.py]: Can this doc string be removed?

File: pulp_python/app/serializers.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
+"""
+Check `Plugin Writer's Guide`_ and `pulp_example`_ plugin
```
Comment (werwty) [inline pulp_python/app/serializers.py]: Can this doc string be removed?

File: pulp_python/app/viewsets.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
+"""
+Check `Plugin Writer's Guide`_ and `pulp_example`_ plugin
```
Comment (werwty) [inline pulp_python/app/viewsets.py]: Can this doc string also be removed?

File: flake8.cfg [hunk 7]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = ./docs/*
+exclude = */migrations/*
```
Comment (werwty) [inline flake8.cfg]: I think we should keep excluding docs

**Pull request merged**